File: csharp/Lift/LiftSystemTest.cs
using System.Collections.Generic;
using Xunit;

namespace Lift
{
    public class LiftSystemTest
    {
        // TODO: enable this test and finish writing it
        // [Fact]
        public void Something()
        {
            var liftA = new Lift("A", 0);
            var lifts = new LiftSystem(new List<int>(){0, 1}, new List<Lift>{liftA}, new List<Call>());
            lifts.Tick();
        }
    }
}

File: kotlin/src/test/kotlin/codingdojo/LiftSystemTest.kt
package codingdojo

import org.junit.jupiter.api.Test

internal class LiftSystemTest {

    @Test
    fun `do something`() {
        val liftA = Lift("A", 0)
        val liftSystem = LiftSystem(floors = listOf(0, 1), lifts = listOf(liftA), calls = emptyList())

        liftSystem.tick()

        // TODO: finish writing this test
    }
}
